docs/specification/objects.md

```diff
@@ -20,6 +20,11 @@ Object Runs associated with an object Spec inherit the [Object Template](../obje
 * Many Runs can reference the same Spec.
 * Each Run must be associated with exactly one Spec.
 
+
+The mechanism for [Attribute](../attributes) validation is through [Attribute Templates](../attribute-templates).  If the Attribute's template is set, the constraints defined in that template will be applied to the Attribute's [Value](../value-types).  If the [Object Template](../object-templates/) has further restricted the bounds of the Attribute Template, then those tighter constraints are enforced.  The constraints defined in the Object Template are applied to an attribute that contains the matching Attribute Template.  
```

Comment:
Please break this into one line per sentence.

Reply:
Is "applied" the most appropriate word ("...applied to the Attribute's Value")? "Applied" makes me think that the constraints are doing something to the value and maybe changing it. Perhaps "checked against" would make more sense? Or "validated against."